README.md

```diff
@@ -12,7 +12,19 @@ Performs a migration from CFME 5.7 to 5.8 utilizing steps from [Migrating to Red
 * Disks have already been resized.
 * Environments are not utilizing database replication.
 
+#### Additionaly required playbooks
+* cfme-repositories.yml
+
 #### Required groups
 * cfme
 * cfme-appliancees
 * cfme-databases
+
+### cfme-repositories.yml
```

Comment:
This isn't a playbook, just a task list, so no need to list since it isn't a "API".